[MRG] fix ClassifierChain issue when a tuple is passed to order (#18124)
scikit-learn/scikit-learn: doc/whats_new/v0.24.rst

  validity of the input is now delegated to the base estimator.
  :pr:`17233` by :user:`Zolisa Bleki <zoj613>`.

:mod:`sklearn.multioutput`
..........................

> - |Fix| A fix to allow :class:`multioutput.ClassifierChain` to accept
>   ``order`` as a tuple.

Reviewer: Maybe a slight improvement,
```suggestion
- |Fix| A fix to accept tuples for the ``order`` parameter 
  in :class:`multioutput.ClassifierChain`.
```
